### DISCUSSION TITLE
Can I achieve customview decoding also in the editor of mitmweb?

### DISCUSSION THREAD

**Discussion by zi0Black** [mitmproxy/mitmproxy#6312] [Q&A] [answered]
I'm using the custom view addon example to implement gRPC-WEB decoding, and it's working okay, but when I click to edit the request/response payload is not parsed and remains in a raw format. Where should I look in the code to ensure the payload is deserialized before I can work on it when I click edit?

Thanks.

**Comment by mhils** [comment, marked as answer]
The contentviews only support pretty-printing at the moment, no re-serialization/round-trips. It's definitely something I'd like us to get to, but it's a significant feature that requires some thought to get right. :)

**Reply by zi0Black** [reply]
Thanks for the answer @mhils
